pkg/odo/cli/delete/delete.go

```diff
@@ -17,6 +17,7 @@ func NewCmdDelete(name, fullName string) *cobra.Command {
 
 	componentCmd := NewCmdComponent(ComponentRecommendedCommandName, util.GetFullName(fullName, ComponentRecommendedCommandName))
 	deleteCmd.AddCommand(componentCmd)
+	deleteCmd.Annotations = map[string]string{"command": "main"}
```

Comment:
It would be nice to have some method to set this value, for example:

```
somepackage.SetCommandSection(deleteCmd, "main")
```

that would create the map if nil, then set the entry for "command".

This would help not completely reassigning the `Annotations` value, as it could already be set with `clientset.Add()`